contracts/exchange/test/assertion_wrappers/fill_order_wrapper.ts

```diff
@@ -24,9 +25,41 @@ import {
 } from '../../src';
 
 export class FillOrderWrapper {
-    private readonly _exchange: ExchangeContract;
     private readonly _blockchainBalanceStore: BlockchainBalanceStore;
 
+    /**
```

Comment:
The following two static methods are not new, but had to be moved up because static members must come first.